libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -127,6 +129,7 @@ export const OcmClusterDetailsFormFields = ({
       const checked = Boolean(value);
       setFieldValue('addCustomManifest', checked, false);
       clusterWizardContext.setAddCustomManifests(checked);
```

Comment:
Disabling the selector is not enough.
If users first select Static IP, and then they select OCI, the selector would become selected and disabled.
We would have a similar case that for the MR where it needs to be disabled based on the OpenShift version.

The new test should also do that, first select Static IP, then select OCI, and verify that DHCP is selected.